content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -13,10 +13,11 @@ def cloud?(prov_type)
 end
 
 def calculate_requested(options_hash = {})
-  {:storage => get_total_requested(options_hash, :storage),
-   :memory  => get_total_requested(options_hash, :vm_memory),
-   :cpu     => get_total_requested(options_hash, :number_of_cpus),
-   :vms     => get_total_requested(options_hash, :number_of_vms)}
+  @total_vms = get_total_requested(options_hash, :number_of_vms)
```

Comment:
@billfitzgerald0120 
Can we just use a total_vms as a local variable and default it to 1 for reconfigure request if its missing.